### ISSUE TITLE
20190324 Github 前端趋势榜

### ISSUE THREAD

**Issue by kujian** [kujian/githubTrending#39]
# 20190324 Github 前端趋势榜
[Github 前端开发日报](http://caibaojian.com/c/news)栏目数据来自[Github热榜](http://news.caibaojian.com/)（我开发的爬虫），每日分享HTML、CSS、JavaScript、Vue等开源项目，为开发者提供动力，点击Star按钮来关注这个项目，点击Watch来收听每日的更新[Github主页](https://github.com/kujian/githubTrending)
## JavaScript
* [leeoniya/dropcss](https://github.com/leeoniya/dropcss) （A simple, thorough and fast unused-CSS cleaner）
* [clone95/Machine-Learning-Study-Path-March-2019](https://github.com/clone95/Machine-Learning-Study-Path-March-2019) （A complete ML study path, focused on TensorFlow and Scikit-Learn）
* [klaussinani/taskbook](https://github.com/klaussinani/taskbook) （&#x1f4d3; Tasks, boards &amp; notes for the command-line habitat）
* [scutan90/DeepLearning-500-questions](https://github.com/scutan90/DeepLearning-500-questions) （深度学习500问，以问答形式对常用的概率知识、线性代数、机器学习、深度学习、计算机视觉等热点问题进行阐述，以帮助自己及有需要的读者。 全书分为15个章节，近20万字。由于水平有限，书中不妥之处恳请广大读者批评指正。 未完待续............ 如有意合作，联系scutjy2015@163.com 版权所有，违权必究 Tan 2018.06）
* [vuejs/vue](https://github.com/vuejs/vue) （        A progressive, incrementally-adoptable JavaScript framework for building UI on the web.      ）
* [alexfoxy/laxxx](https://github.com/alexfoxy/laxxx) （Simple &amp; light weight vanilla javascript plugin to create beautiful animations things when you scrolllll!! Harness the power of the most intuitive interaction and make your websites come alive!）
* [syhyz1990/baiduyun](https://github.com/syhyz1990/baiduyun) （油猴脚本 直接下载百度网盘和百度网盘分享的文件,直链下载超级加速）
* [facebook/react](https://github.com/facebook/react) （A declarative, efficient, and flexible JavaScript library for building user interfaces.）
* [nytimes/library](https://github.com/nytimes/library) （A collaborative documentation site, powered by Google Docs.）
* [Ibit-to/google-unlocked](https://github.com/Ibit-to/google-unlocked) （Google Unlocked Chrome extension unlocks hidden google search results）
* [Coalfire-Research/npk](https://github.com/Coalfire-Research/npk) （A mostly-serverless distributed hash cracking platform）
* [lyricat/wechat-format](https://github.com/lyricat/wechat-format) （微信公众号排版编辑器，Markdown 到给微信特制的 HTML）
* [nature1995/ran-django-template](https://github.com/nature1995/ran-django-template) （&#x1f525;An awesome Django project template for web include restful api | Author: Ziran Gong）
* [zhihu/griffith](https://github.com/zhihu/griffith) （A React-based web video player）
* [LisaDziuba/Awesome-Design-Tools](https://github.com/LisaDziuba/Awesome-Design-Tools) （The best design tools for everything.）
* [Authman2/Mosaic](https://github.com/Authman2/Mosaic) （&#x1f3a8; A front-end JavaScript library for building user interfaces.）
* [trekhleb/javascript-algorithms](https://github.com/trekhleb/javascript-algorithms) （Algorithms and data structures implemented in JavaScript with explanations and links to further readings）
* [iamkun/dayjs](https://github.com/iamkun/dayjs) （⏰ Day.js 2KB immutable date library alternative to Moment.js with the same modern API）
* [30-seconds/30-seconds-of-code](https://github.com/30-seconds/30-seconds-of-code) （Curated collection of useful Javascript snippets that you can understand in 30 seconds or less.）
* [GoogleChrome/puppeteer](https://github.com/GoogleChrome/puppeteer) （Headless Chrome Node API）
* [mozilla/send](https://github.com/mozilla/send) （File Sharing Experiment）
* [nodejs/node](https://github.com/nodejs/node) （        Node.js JavaScript runtime ✨??✨      ）
* [facebook/create-react-app](https://github.com/facebook/create-react-app) （Create React apps with no build configuration.）
* [mrdoob/three.js](https://github.com/mrdoob/three.js) （        JavaScript 3D library.      ）
* [gatsbyjs/gatsby](https://github.com/gatsbyjs/gatsby) （⚛️?? Blazing fast static site generator for React）
* [brandonweiss/charge](https://github.com/brandonweiss/charge) （⚡️ An opinionated, zero-config static site generator.）
* [iview/iview-admin](https://github.com/iview/iview-admin) （Vue 2.0 admin management system template based on iView）

## Vue
* [teamprevue/PreVue](https://github.com/teamprevue/PreVue) （&#x1f3a8; Single page application prototyping tool for developers and designers.）
* [emilkowalski/css-effects-snippets](https://github.com/emilkowalski/css-effects-snippets) （A collection of CSS effects made with Vue.js.）
* [dcloudio/uni-app](https://github.com/dcloudio/uni-app) （使用 Vue.js 开发跨平台应用的前端框架）
* [PanJiaChen/vue-element-admin](https://github.com/PanJiaChen/vue-element-admin) （        vue2 admin / a management system template      ）
* [creativecommons/cccatalog-frontend](https://github.com/creativecommons/cccatalog-frontend) （CC Search Beta is a new search frontend for CC-licensed works across the internet.）
* [liuzhuoling2011/baidupcs-web](https://github.com/liuzhuoling2011/baidupcs-web) （基于vue2+nuxt构建的高仿饿了么（2018版））
* [geektime-geekbang/geektime-vue-1](https://github.com/geektime-geekbang/geektime-vue-1) （极客时间基础篇代码）
* [iview/iview](https://github.com/iview/iview) （        A high quality UI Toolkit with both Vue.js 2.0 and 1.0      ）
* [vueComponent/ant-design-vue](https://github.com/vueComponent/ant-design-vue) （An enterprise-class UI components based on Ant Design and Vue. ?）
* [vuematerial/vue-material](https://github.com/vuematerial/vue-material) （        Material design for Vue.js      ）
* [jbaysolutions/vue-grid-layout](https://github.com/jbaysolutions/vue-grid-layout) （A draggable and resizable grid layout, for Vue.js.）
* [salomonelli/best-resume-ever](https://github.com/salomonelli/best-resume-ever) （? ? Build fast ? and easy multiple beautiful resumes and create your best CV ever! Made with Vue and LESS.）

## HTML
* [changtuiqie/WeChatAgreement.WebApi.Simple](https://github.com/changtuiqie/WeChatAgreement.WebApi.Simple) （mac微信,ipad微信 webapi封装好的实现方案，免IIS一键部署。 可实现微信80%功能；支持62数据登录、扫码登录、收发朋友圈、查看朋友圈、微信建群、微信拉人进群、微信公众号阅读、微信消息收发、微信附近的人定位、微信添加好友、微信红包接收、微信防撤回、分享小程序、微信加粉、微信收藏、微信标签等）
* [RemoteML/bestofml](https://github.com/RemoteML/bestofml) （The best resources around Machine Learning）
* [Alivon/Panda-Learning](https://github.com/Alivon/Panda-Learning) （熊猫学习Panda-Learning是一个辅助学习强国的软体,帮助挤不出时间，却仍然需要学习的积极分子，盼达学习强国(xuexiqiangguo)）
* [hamukazu/lets-get-arrested](https://github.com/hamukazu/lets-get-arrested) （This project is intended to protest against the police in Japan）
* [google/styleguide](https://github.com/google/styleguide) （        Style guides for Google-originated open-source projects      ）
* [facebookresearch/fastText](https://github.com/facebookresearch/fastText) （Library for fast text representation and classification.）
* [iliakan/javascript-tutorial-en](https://github.com/iliakan/javascript-tutorial-en) （        Modern JavaScript Tutorial       ）
* [30-seconds/30-seconds-of-css](https://github.com/30-seconds/30-seconds-of-css) （A curated collection of useful CSS snippets.）
* [QSCTech/zju-icicles](https://github.com/QSCTech/zju-icicles) （浙江大学课程攻略共享计划）
* [kubernetes-csi/docs](https://github.com/kubernetes-csi/docs) （Documentation for CSI integration with Kubernetes）
* [portainer/portainer](https://github.com/portainer/portainer) （        Simple management UI for Docker      ）
* [uikit/uikit](https://github.com/uikit/uikit) （A lightweight and modular front-end framework for developing fast and powerful web interfaces）
* [flutterchina/flutter-in-action](https://github.com/flutterchina/flutter-in-action) （《Flutter实战》电子书）
* [pure-css/pure](https://github.com/pure-css/pure) （A set of small, responsive CSS modules that you can use in every web project.）
* [cloudfavorites/favorites-web](https://github.com/cloudfavorites/favorites-web) （Cloud collection, making the collection easier）
* [greggman/webgl-fundamentals](https://github.com/greggman/webgl-fundamentals) （WebGL lessons that start with the basics）
* [google/WebFundamentals](https://github.com/google/WebFundamentals) （        Best practices for modern web development      ）
* [edent/SuperTinyIcons](https://github.com/edent/SuperTinyIcons) （Under 1KB each! Super Tiny Icons are miniscule SVG versions of your favourite website and app logos）
* [kujian/frontendDaily](https://github.com/kujian/frontendDaily) （前端开发技术日报，每日分享互联网最精彩的前端技术、前端资讯、后端编程、设计和资源等，欢迎关注Watch）
* [fluent-ffmpeg/node-fluent-ffmpeg](https://github.com/fluent-ffmpeg/node-fluent-ffmpeg) （A fluent API to FFMPEG (http://www.ffmpeg.org)）
* [zeit/now-github-starter](https://github.com/zeit/now-github-starter) （Starter project to demonstrate a project whose pull requests get automatically deployed）
* [octocat/Spoon-Knife](https://github.com/octocat/Spoon-Knife) （        This repo is for demonstration purposes only.      ）

## CSS
* [nikitavoloboev/my-mac-os](https://github.com/nikitavoloboev/my-mac-os) （? A list of applications, alfred workflows and various tools that make my macOS experience even more amazing）
* [github/personal-website](https://github.com/github/personal-website) （Code that'll help you kickstart a personal website that showcases your work as a software developer.）
* [daneden/animate.css](https://github.com/daneden/animate.css) （        A cross-browser library of CSS animations. As easy to use as an easy thing.      ）
* [jgthms/bulma](https://github.com/jgthms/bulma) （        Modern CSS framework based on Flexbox      ）
* [zhaoolee/StarsAndClown](https://github.com/zhaoolee/StarsAndClown) （☀️Github星聚弃疗榜,为Github优秀项目写一封推荐信，让Github优秀项目造福人类~ Github apocalypse list, write a letter of recommendation for Github's outstanding projects, and let Github excellent projects benefit humanity~）
* [tailwindcss/tailwindcss](https://github.com/tailwindcss/tailwindcss) （A utility-first CSS framework for rapid UI development.）
* [necolas/normalize.css](https://github.com/necolas/normalize.css) （A collection of HTML element and attribute style-normalizations      ）
* [theme-next/hexo-theme-next](https://github.com/theme-next/hexo-theme-next) （Elegant and powerful theme for Hexo.）
* [rstacruz/cheatsheets](https://github.com/rstacruz/cheatsheets) （My cheatsheets）
* [nostalgic-css/NES.css](https://github.com/nostalgic-css/NES.css) （NES-style CSS Framework | ファミコン風CSSフレームワーク）
* [iissnan/hexo-theme-next](https://github.com/iissnan/hexo-theme-next) （        Elegant theme for Hexo.       ）
* [barryclark/jekyll-now](https://github.com/barryclark/jekyll-now) （        Build a Jekyll blog in minutes, without touching the command line.      ）
* [StylishThemes/GitHub-Dark](https://github.com/StylishThemes/GitHub-Dark) （         Dark GitHub style      ）
* [LiangJunrong/document-library](https://github.com/LiangJunrong/document-library) （jsliang 的文档库. 里面包含了所有的前端文章，例如 vue、react,、angular、微信小程序、设计模式等……）
* [picturepan2/spectre](https://github.com/picturepan2/spectre) （        Spectre.css - a lightweight, responsive and modern CSS framework.      ）
* [carloscuesta/gitmoji](https://github.com/carloscuesta/gitmoji) （        An emoji guide for your commit messages. ?       ）
* [Requarks/wiki](https://github.com/Requarks/wiki) （Wiki.js | A modern, lightweight and powerful wiki app built on NodeJS, Git and Markdown）
* [udacity/course-collaboration-travel-plans](https://github.com/udacity/course-collaboration-travel-plans) （?Jalpc -- A flexible Jekyll theme, 3 steps to build your website.）

## TypeScript
* [codercom/code-server](https://github.com/codercom/code-server) （Run VS Code remotely.）

## Unknow
* [ripienaar/free-for-dev](https://github.com/ripienaar/free-for-dev) （        A list of SaaS, PaaS and IaaS offerings that have free tiers of interest to devops and infradev      ）

## 关于我
 [前端开发博客](http://caibaojian.com/) 站长，[微博](https://weibo.com/kujian) 粉丝18万，[微信公众号](http://t.cn/EV7pcM1)。
如果你觉得我的分享对你有帮助，不妨考虑打赏我[一杯咖啡](http://caibaojian.com/donation)，以资鼓励。
![打赏](https://upload-images.jianshu.io/upload_images/570843-db4053c67a8c9ea9.png)
## 推荐
1. [掘金小册8折优惠，还有全网优惠教程](http://caibaojian.com/goods)
2. [36个小而美的赚钱项目，让你月入10万](http://t.cn/EV7YgYw)
## 微信群
加入微信群，每天晚上我会汇总一天最好的文章给你。
![qianduanquan](https://user-images.githubusercontent.com/3055447/38468989-651132ac-3b80-11e8-8e6b-15122322a9d7.png)
